main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
@towardthesea I eventually concur with @Tobias-Fischer that this isn't really necessary.
For example, in our `IOL` we don't use any fallback value in the configuration file for the table height.

I can see your concern and I appreciate a lot your attempt to make things safer 😄, but most of the times we should favour the [**KISS**](https://en.wikipedia.org/wiki/KISS_principle) principle when we can reasonably guarantee that the system components are launched and well connected (i.e. `ARE` in this case).